addons/breathing/stringtable.xml

```diff
@@ -531,21 +520,7 @@
             <Finnish>1x hemopneumotoraksi</Finnish>
         </Key>
         <Key ID="STR_KAT_Breathing_tensionpneumothorax_display">
-            <English>Needle Decompression</English>
-            <German>Nadel Dekompression</German>
-            <Spanish>Descompresión con aguja</Spanish>
-            <Polish>Igła dekompresyjna</Polish>
-            <Chinese>针减压</Chinese>
-            <Chinesesimp>胸腔穿刺减压</Chinesesimp>
-            <Czech>Dekompresní Jehla</Czech>
-            <Korean>바늘 감압</Korean>
-            <French>Aiguille de décompression</French>
-            <Turkish>İğne Dekompresyonu</Turkish>
-            <Italian>Decompressione con ago</Italian>
-            <Russian>Декомпрессия иглы</Russian>
-            <Japanese>胸腔穿刺減圧</Japanese>
-            <Portuguese>Descompressão com Agulha</Portuguese>
-            <Finnish>Neulan dekompressio</Finnish>
+            <English>Finger Thoracostomy (AAT Kit)</English>
```

Comment:
I agree with Max here, also you need a scalpel to cut open the thorax and insert the tube. Guess this change is good.